src/Categories/Diagram/Cocone/Properties.agda

```diff
@@ -45,6 +46,15 @@ module _ {F : Functor J C} (G : Functor C D) where
     }
     where open CF.Cocone⇒ f
 
+  F-map-Functorˡ : Functor (Cocones F) (Cocones (G ∘F F))
+  F-map-Functorˡ = record
+    { F₀ = F-map-Coconeˡ
+    ; F₁ =  F-map-Cocone⇒ˡ
+    ; identity = λ {A} → G.identity
+    ; homomorphism = λ {X} {Y} {Z} {f} {g} → G.homomorphism
```

Comment:
here (and everywhere in this PR), can you delete implicits if/when they are not needed please?

Reply:
Sure! I've removed all implicit arguments from the lambdas in my commit.